### PR TITLE
#11 - Visual Chart - George Primentas

### LINKED ISSUE #11: Visual Chart
| name  | about | title  | labels | assignees  |
| ------------- | ------------- | ------------- | ------------- | ------------- |
| "👥 User story"  | Create a new user story  | VISUAL CHART | Feature| TBC  | 

## Story: 
*As a ....user,
I want to..... be able to see a visual chart that shows everyones pull request contributions
So that I .....can see and know everyones contribution.

#Acceptance criteria:

1. GIVEN ... I am on the app’s landing page
- WHEN ... I click into a particular team/group card,
- THEN ... I should be able to see a visual chart in form of a type of graph representation, showing each team member PR and contributions Ito the project.

## Checklist:

- [ ] I have defined the user story
- [ ] I have defined some acceptance criteria
- [ ] I have included any implementation notes
- [ ] I have included all relevant links

### PR DESCRIPTION
**Pull request by GeorgePrimentas**
Component for the Team card page where the percentage of PR contribution of each member is represented in a hundred-square chart.

Fixes #11 

Checklist:

    My code follows the style guidelines of this project
    I have carefully reviewed my own code
    I have commented my code
    I have updated any documentation

### CODE REVIEW

File: client/src/components/HundredSquareChart/HundredSquareChart.jsx [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+      .on("mouseout", function () {
+        d3.select(this).attr("stroke", "none");
+        tooltip.style("visibility", "hidden");
+      });
+
+  }, [data]);
+
+  return (
+    <div>
+      <p>{JSON.stringify(data)}</p>
```
Comment (AndrianaOS) [inline client/src/components/HundredSquareChart/HundredSquareChart.jsx]: do we really need this line. The table will show the Pull request numbers

**Review by AndrianaOS — CHANGES_REQUESTED** [review]
This is amazing work George. I was going to ask could some of the styles go into the CSS but i dont know how building this works.
if you use this fetch url: (http://localhost:8000/api/members) which is the getAllTeamsAndMembersData you should be able to get the data you need to get create the graph so you dont have to hardcode the information in.

File: client/src/components/HundredSquareChart/HundredSquareChart.jsx (hunk 1 above)
Comment (Gayle-Thompson-Igwebike) [reply client/src/components/HundredSquareChart/HundredSquareChart.jsx]: I cant view your lovely work George, even though I have installed d3* ... anyways, when we meet, you can share with us. Well done!

**Comment by AndrianaOS** [comment]
http://localhost:8000/teamInfo/${teamID} for specific teams or 
http://localhost:8000/teamPr for all teams data

**Review by AndrianaOS — APPROVED** [review]

**Pull request merged**